index.html

```diff
@@ -13461,6 +13453,69 @@ <h2>Including Elements in the Accessibility Tree</h2>
             </li>
           </ul>
 	</section>
+	<section id="tree_relationships">
+	  <h2>Relationships in the Accessibility Tree</h2>
+          <p>The following terms are used to describe relationships between <abbr title="Document Object Model">DOM</abbr> elements.</p>
+          <p>The <dfn data-export="" data-lt="accessibility child|owned child|child element|child|children|child elements">accessibility children</dfn> of a DOM element are all of the following (excluding those DOM elements that have no corrosponding <a>accessible object</a> because they have been <a href="#tree_exclusion">excluded from the accessibility tree<a>, and excluding any elements whose corrosponding <a>accessible object</a> have been reparented in the <a>accessibility tree</a> via <pref>aria-owns</pref>):</p>
+          <ul>
+            <li>The <abbr title="Document Object Model">DOM</abbr> children of the <a>element</a>.
+            </li>
+            <li>All descendant of the <a>element</a> with only elements of role <rref>generic</rref> or <rref>none</rref> intervening.
```

Comment:
```suggestion
            <li>All descendants of the <a>element</a> with only elements of role <rref>generic</rref> or <rref>none</rref> intervening.
```
Spelling suggestion.